<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStaffWindow</class>
 <widget class="QDialog" name="EditStaffWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="AgeLabel_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9Name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13Gender">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Staff Information:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="NamepushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GenderpushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update gender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AgepushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13ID2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonID">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPass">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update password</string>
   </property>
  </widget>
  <widget class="QLabel" name="PassLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Admin Dashboard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
